### PR TITLE
Add PowerShell prerequisites and http lesson

### PR DESCRIPTION
**Pull request by Ba4bes**
Starting of for #16 : 

- Adds PowerShell HTTP lesson
- Adds PowerShell HTTP homework
- Adds example functions for both lesson and homework
- Adds references to lessons/_index.md

### CODE REVIEW

File: lessons/PowerShell/http/http-homework-powershell.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# HTTP Trigger Homework, make a resume API (.NET Core)
```
Comment (marcduiker) [inline lessons/PowerShell/http/http-homework-powershell.md:1]:
```suggestion
# HTTP Trigger Homework, make a resume API (PowerShell)
```

File: lessons/PowerShell/http/http-lesson-powershell.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+Let's change the template to find out what parameters can be changed.
+Depending on the trigger, arguments can be added/removed and parameter types can be changed.
+Start with only allowing GET requests.
+
+### Steps
+
+1. Open function.json. In the httpTrigger binding, the `methods` are listed. Remove `"post"`. Don't forget to remove the comma after `"get"`. Now the function can only be triggered by a GET request.
+
+    > 🔎 **Observation** - You'll notice that this change breaks the code inside the function. This is because the `HttpRequestMessage` type has different properties and methods than the `HttpRequest` type.
```
Comment (marcduiker) [inline lessons/PowerShell/http/http-lesson-powershell.md:79]: I didn't test the exercise myself just yet, but is this observation correct here? I don't see any mention of changing the request type in step 1 (?).

File: lessons/_index.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ We currently have these lessons planned and we'll be adding more content over ti
 
 |Nr|Description|Language specific lesson|Stream/Recording|Contributions by
 |-|-|-|-|-
-|0|Prerequisites|[.NET Core](prerequisites-dotnet.md) , [TypeScript](prerequisites-ts.md)|[Oct 28th](https://youtu.be/5k35dlBAXxA)|Marc & Gwyneth (.NET Core) , Christian & Marc (TypeScript)
-|1|HTTP Trigger|[.NET Core](dotnet/http/http-lesson-dotnet.md) , [TypeScript](http-ts.md)|[Oct 28th](https://youtu.be/5k35dlBAXxA)|Marc & Gwyneth (.NET Core) , Christian & Marc (TypeScript)
+|0|Prerequisites|[.NET Core](prerequisites-dotnet.md) , [PowerShell](prerequisites-powershell.md), [TypeScript](prerequisites-ts.md)|[Oct 28th](https://youtu.be/5k35dlBAXxA)|Marc & Gwyneth (.NET Core) , Barbara (PowerShell), Christian & Marc (TypeScript)
```
Comment (marcduiker) [inline lessons/_index.md:17]: I've simplified this table and dropped the column with the direct links to the YouTube videos (so there's a merge conflict now 😬). I recommend you pull the latest changes and re-apply yours.

File: lessons/prerequisites-powershell.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Prerequisites
```
Comment (marcduiker) [inline lessons/prerequisites-powershell.md:1]:
```suggestion
# Prerequisites (PowerShell)
```

**Review by marcduiker — CHANGES_REQUESTED** [review]
This looks great! Excellent tips you've provided. And the reference of 📎 in the homework!

Only some minor additions, questions (see the details) and a merge conflict with the md table to be fixed (sorry 😅).

File: lessons/PowerShell/http/http-homework-powershell.md (hunk 1 above)
Comment (Ba4bes) [reply lessons/PowerShell/http/http-homework-powershell.md]: Missed that one, fixed!

File: lessons/PowerShell/http/http-lesson-powershell.md (hunk 2 above)
Comment (Ba4bes) [reply lessons/PowerShell/http/http-lesson-powershell.md]: The request type was changed by changing the json, but that did not break the function. Removed the observation

File: lessons/_index.md (hunk 3 above)
Comment (Ba4bes) [reply lessons/_index.md]: well that was a fun merge 😀.
Added the links to the new table

File: lessons/prerequisites-powershell.md (hunk 4 above)
Comment (Ba4bes) [reply lessons/prerequisites-powershell.md]: Added

**Review by marcduiker — APPROVED** [review]
LGTM, thank you!

**Pull request merged**